apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/(analysis)/summary/components/shareEmbedTabs/EmailTab.tsx

```diff
@@ -0,0 +1,396 @@
+import { sendEmailAction } from "@/app/(app)/environments/[environmentId]/surveys/[surveyId]/(analysis)/summary/actions";
+import CodeBlock from "@/components/shared/CodeBlock";
+import { cn } from "@formbricks/lib/cn";
+import { QuestionType } from "@formbricks/types/questions";
+import { TProduct } from "@formbricks/types/v1/product";
+import { TProfile } from "@formbricks/types/v1/profile";
+import { TSurvey } from "@formbricks/types/v1/surveys";
+import { Button, Input } from "@formbricks/ui";
+import { CodeBracketIcon, DocumentDuplicateIcon, EnvelopeIcon } from "@heroicons/react/24/solid";
+import {
+  Column,
+  Container,
+  Button as EmailButton,
+  Link,
+  Row,
+  Section,
+  Tailwind,
+  Text,
+  render,
+} from "@react-email/components";
+import { useMemo, useState } from "react";
+import toast from "react-hot-toast";
+
+interface EmailTabProps {
+  survey: TSurvey;
+  surveyUrl: string;
+  profile: TProfile;
+  product: TProduct;
+}
+
+export default function EmailTab({ survey, surveyUrl, profile, product }: EmailTabProps) {
+  const [email] = useState(profile.email);
+  const [showEmbed, setShowEmbed] = useState(false);
+  const brandColor = product.brandColor;
```

Comment:
you're only using the `brandColor` from the product, I think you can just pass the `brandColor` directly in this component, same with `email` from the `profile` 😊